types/getConfigurationFromServer.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Configuration options for AWS Runtimes
+ * Sent back to the client to signal available configuration values
+ */
+export interface ConfigurationOptions {
```

Comment:
Does this object really need to be in `types` package and not `protocol`?

Reply:
I was also conflicted, seeing as `ChatOptions` is also defined in types https://github.com/aws/language-server-runtimes/blob/14c7fc6c782faa7cf22f39d889f659d4faa7c8a6/runtimes/protocol/lsp.ts#L109 
I wanted to keep consistency so I moved it to types

Reply:
Yeah, I'd keep scope of changes small tbh, and don't move stuff to types unnecessarily. It was originally meant to host types around Chat feature. Maybe keeping LSP `Options` types there is not right and we'll move `ChatOptions` out as well.